docs/Buttons.md

```diff
@@ -7,11 +7,33 @@ title: "Buttons"
 
 React-Admin provides button components for all the common uses.
 
-## Navigation Buttons
-
-These buttons allow users to navigate between the various react-admin views.
-
-### `<EditButton>`
+- **Navigation Buttons**: to navigate between the various react-admin views.
```

Comment:
I removed the sub sections ('navigation', 'list', etc) from the content and added this TOC instead to move all the section levels up (`####`=> `###`). This makes the right menu much more useful.